SonarQube.Bootstrapper/Program.cs

```diff
@@ -30,24 +29,22 @@ public static int Execute(string[] args, IBuildAgentUpdater updater, ILogger log
             IBootstrapperSettings settings;
             if (ArgumentProcessor.TryProcessArgs(args, logger, out settings))
             {
+                Debug.Assert(settings != null, "Bootstrapper settings should not be null");
                 Debug.Assert(settings.Phase != AnalysisPhase.Unspecified, "Expecting the processing phase to be specified");
 
-                string[] strippedArgs = ArgumentProcessor.RemoveBootstrapperArgs(args);
-                string cmdLineParams = GetChildProcCmdLineParams(strippedArgs);
```

Comment:
I've moved the calculation the args to pass to the child process to the ArgumentProcessor.